WcaOnRails/config/locales/en.yml

```diff
@@ -1302,3 +1303,79 @@ en:
+    panel3:
+      title: "Regulations FAQ"
+      content1_html: 'The <a href="/faq">FAQ section</a> answers several frequently asked questions about the WCA Regulations.'
+    panel4:
+      title: "Organizer Guidelines"
+      content1_html: 'If you are interested in organizing a competition or already planning to organize one, please read our <a href="/organizer_guidelines">Organizer Guidelines</a>.'
+      content2_html: 'This document contains important information and should be read by every competition organizer.'
+  #context: Organizer Guidelines
+  organizer_guidelines:
+    title: "WCA Competition Organizer Guidelines"
+    paragraph1:
+      title: "How to organize a competition"
+      content: "All WCA competitions must be attended and overseen by a WCA Delegate. If you are interested in organizing your own competition, please contact your nearest WCA Delegate. They will be able to give you more information on the local practices and can
+      help you with finding a suitable date as they know if there are any other competitions planned in the same time frame. If there is no WCA Delegate near your area, please get in touch with the Senior Delegate for your region."
+    paragraph2:
+      title: "Expected expenses for organizing a competition"
+      content: "The expected expenses differ greatly from country to country and are determined by several factors like the size of the venue and other general conditions. Recouping all expenses with entry fees is possible. However, to keep these as low as possible,
+      it is highly recommended to search for sponsors. Although WCA Delegates are volunteers and do not receive a salary for their work, organizers should consider paying compensation for their travel expenses."
+    paragraph3:
+      title: "Deadlines for the announcement of the competition"
+      content: "The competition should be announced at least one month before the start of the competition (see Guideline 8a4++). If there are special reasons preventing this deadline to be met, competitions may be announced up to two weeks before, at the discretion
+      of the WCA Board. It is highly recommended to inform the WCA Board of these reasons as soon as possible to avoid a later refusal of the competition."
+    paragraph4:
+      title: "Proximity policy"
+      content1: "As a general policy, WCA Competitions will be accepted if they are at least 200 km or 26 days away from any other competition. WCA Board should particularly review a proposed competition if there is another competition for which the distance between
+      them is less than 200 km and the time between them is less than 27 days. A delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that will support such
```

Comment:
To me, "at least 200km or 26 days" means `>= 200 km or >= 26 days`, and the inverse of that is `< 200 km and < 26 days`.